examples/14-community-hosp/main.cpp

```diff
@@ -0,0 +1,133 @@
+#include "../../epiworld.hpp"
+
+/***
+ * A concrete example would be a model that includes two populations.
```

Comment:
Should be reworded to "A concrete example with a model that includes two populations:"

Reply:
I'm addressing this in #34.